CompetitionTask/StepDefinitions/MarsEducationCertificationsTestsStepDefinitions.cs

```diff
@@ -0,0 +1,269 @@
+using Mars_Education_Certifications.Pages;
+using MarsEducationCertificationsSpecflow.Pages;
+using MarsEducationCertificationsSpecflow.Utilities;
+using OpenQA.Selenium;
+using AventStack.ExtentReports.Gherkin.Model;
+using MarsEducationCertificationsSpecflow.JSONDataObject;
+using MarsEducationCertificationsTests.JSONDataObject;
+using System.Diagnostics.Metrics;
+using MarsEduCertAutomation.Utilities;
+using NUnit.Framework;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using System.Collections.Generic;
+using MarsEduCertAutomation.Utility;
+using NUnit.Framework.Interfaces;
+
+
+namespace MarsEducationCertificationsSpecflow.StepDefinitions
+{
+  
+    [TestFixture]
+    public class MarsEducationCertificationsTestsStepDefinitions
+    {
+
+        // private IWebDriver _commonDriver;
+        private IWebDriver _driver;
+        private CommonDriver _commonDriver;
+        private LoginPage _loginPageObj;
+        private ProfilePage _profilePageObj;
+        private EducationFeature _educationFeatureObj;
```

Comment:
can you also split out your tests into two test files one for education and one for certification

Reply:
Resolved